releasing.md

```diff
@@ -0,0 +1,304 @@
+# Releasing AeroGear Digger Java Client
+
+### Big Picture
+
+* We release to Sonatype Nexus.
+* That syncs with Maven Central.
+* We don't use JBoss Nexus as it is slower and there is no need for that if we are able to publish to Sonatype.
```

Comment:
remove that sentence - not need to include this as a potential polemic sentence ...